src/ConfigProvider.php

```diff
@@ -4,38 +4,31 @@
 
 namespace Answear\GlsBundle;
 
-use Answear\GlsBundle\Enum\CountryCode;
+use Answear\GlsBundle\Enum\CountryCodeEnum;
 
 class ConfigProvider
 {
-    private const URL_MAP = [
-        CountryCode::HUNGARY => 'https://online.gls-hungary.com/',
-        CountryCode::SLOVAKIA => 'https://online.gls-slovakia.com/',
-        CountryCode::CZECH => 'https://online.gls-czech.com/',
-        CountryCode::ROMANIA => 'https://online.gls-romania.com/',
-        CountryCode::SLOVENIA => 'https://online.gls-slovenia.com/',
-        CountryCode::CROATIA => 'https://online.gls-croatia.com/',
-    ];
-
-    private CountryCode $countryCode;
+    private CountryCodeEnum $countryCode;
 
     public function __construct(string $countryCode)
     {
-        $this->countryCode = CountryCode::byValue($countryCode);
+        $this->countryCode = CountryCodeEnum::from($countryCode);
     }
 
-    public function getCountryCode(): CountryCode
+    public function getCountryCode(): CountryCodeEnum
     {
         return $this->countryCode;
     }
 
     public function getUrl(): string
     {
-        $url = self::URL_MAP[$this->getCountryCode()->getValue()] ?? null;
-        if (null === $url) {
-            throw new \InvalidArgumentException('No url for provided country code.');
-        }
-
-        return $url;
+        return match ($this->getCountryCode()) {
+            CountryCodeEnum::Romania,
```

Comment:
Why romania has changed url from `https://online.gls-romania.com/`

Reply:
`https://online.gls-romania.com/` is an old endpoint and is not supported by the current changes in this PR.
Romania does not have its own new endpoint, similar to other countries. Instead, we can use the following endpoint to retrieve the list of parcel pickup points: `https://map.gls-hungary.com/data/deliveryPoints/ro.json`